### PR TITLE
Added django 1.10 support

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
         'aldryn-reversion>=1.0.9',
         'django-treebeard>=4.0.1',         # django-cms
         'djangocms-admin-style>=1.2.0',    # django-cms
-        'django-select2>=4.3,<5.0',
+        'django-select2>=4.3',
```
Comment (stefanfoulis) [inline setup.py:24]: is this correct?

**Review by stefanfoulis — APPROVED** [review]

File: aldryn_django_cms/views.py [hunk 2]
```diff
@@ -66,9 +66,13 @@ def check_uninstall_ok(request):
...
         # generator of classes found in menu.py module
-        cms_menu_classes = get_classes_from_module(app=app, module_name="menu")
+        old_cms_menu_classes = set(get_classes_from_module(app=app, module_name="menu"))
 
-        for menu_class in cms_menu_classes:
+        # 3.4 supports but does not require cms_menus module
+        # will be required in 3.5
+        new_cms_menu_classes = set(get_classes_from_module(app=app, module_name="cms_menus"))
+
+        for menu_class in (old_cms_menu_classes|new_cms_menu_classes):
```
Comment (GaretJax) [inline aldryn_django_cms/views.py:75]: Using `set` looses ordering; does this have any influence on how the menus get loaded?

File: setup.py (hunk 1 above)
Comment (GaretJax) [inline setup.py:24]: You probably know better, but is this really the case?

**Review by GaretJax — APPROVED** [review]
Please take a look at the comments; I'm in no position to judge, and you probably know better.

File: aldryn_django_cms/views.py (hunk 2 above)
Comment (GaretJax) [reply aldryn_django_cms/views.py:75]: Also, spacing around operators :-P

Comment (czpython) [reply aldryn_django_cms/views.py:75]: None :)
This is run only when an application is about to be uninstalled through control panel.

Comment (GaretJax) [reply aldryn_django_cms/views.py:75]: 👍

**Comment by czpython** [comment]
@stefanfoulis @GaretJax 
Regarding select2, versions under 5.0 don't support django 1.10.
As a result, is needed to relax the version here.

This will likely cause problems for people upgrading their projects which relied on select2 since 5.0 was a breaking release, this could also affect some of our addons.

The goal of this PR is to get django-cms blank project to work, the rest (xplorer and upgrades) we'll need to address separately. /cc @FinalAngel

**Pull request merged**